CONTRIBUTING.md

```diff
@@ -0,0 +1,54 @@
+## Requirements
+
+We will explain how to contribute to this project using a linux machine, in order to be able ot easly contribute you need:
+
+ - A machine with a public IP in order to use let's encrypt (you can provision ad-hoc machine on any cloud provider you use, or port-forward your home network for dev and testing)
+- A domain with A and/or AAAA records set to your server's IP
+ - [kind](https://github.com/kubernetes-sigs/kind) (to create a development cluster)
+ - [skaffold](https://skaffold.dev/) (to improve development experience)
+ - [Docker HUB](https://hub.docker.com) account (to store your docker images)
+
+## Setup a development cluster
```

Comment:
This is fine but should probably make it explicit that it is optional. Kubernetes comes with docker for Mac so I don’t want someone getting confused and setting up kind if they don’t need to 😛

Reply:
@dannav It's fine for you add a note like:

> OPTIONAL: this is needed only if you don't have a kubernetes cluster to use for development purpose (like Docker for Mac)

Reply:
@mavimo That looks good to me, sure -- please go for it!